test/prettier.mjs

```diff
@@ -126,11 +72,12 @@ ruleTester.run('prettier', rule, {
       code: `('');\n`,
       filename: getPrettierRcJsFilename('single-quote', 'dummy.md'),
     },
-    // Should ignore files from node_modules
-    {
-      code: 'a();;;;;;\n',
-      filename: 'node_modules/dummy.js',
-    },
+    // TODO: enable this one https://github.com/eslint/eslint/issues/19471 get fixed
```

Comment:
Can we enable this for ESLint v8?

Reply:
https://github.com/prettier/eslint-plugin-prettier/pull/709/commits/d1de552acf0d5fc2a2972390c620aa7b225904eb